apps/playground/src/app/quiz/containers/question/question.component.ts

```diff
@@ -0,0 +1,256 @@
+import { Component, OnInit, Input, Output } from '@angular/core';

```

Comment:
this file seems like it's smth like `dependency-injection-quiz-component` or smth like this?

Reply:
ok made this change

Reply:
ok changed the tag in the template to dependency-injection-quiz, but getting this message in WebStorm: "property question is not provided by any applicable directives nor by dependency-injection-quiz element"

Reply:
ok I resolved this somehow